pkg/containerruntime/crio/supported_versions.go

```diff
@@ -0,0 +1,26 @@
+package crio
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver"
+	"github.com/kubermatic/machine-controller/pkg/errors"
+)
+
+// GetOfficiallySupportedVersions returns the officially supported cri-o version for the given kubernetes version
+func GetOfficiallySupportedVersions(kubernetesVersion string) ([]string, error) {
+	v, err := semver.NewVersion(kubernetesVersion)
+	if err != nil {
+		return nil, err
+	}
+
+	majorMinorString := fmt.Sprintf("%d.%d", v.Major(), v.Minor())
+	switch majorMinorString {
```

Comment:
Wouldn't be a `strings.HasPrefix(kubernetesVersion, "1.8")` be sufficient? But I don't have a problem with using this lib.

Reply:
It prevents breaking the code when someone enters `v1.9.2` or `1.9.2`

Reply:
Fair enough. Then using that lib is probably better then starting your own regex magic. :sparkles: